src/contracts/ChainContract.ts

```diff
@@ -1,13 +1,15 @@
 import {inject, injectable} from 'inversify';
-import {Contract} from 'ethers';
+import {Contract, BigNumber} from 'ethers';
 import {TransactionResponse} from '@ethersproject/providers';
-import {ABI, ContractRegistry} from '@umb-network/toolbox';
+import {ContractRegistry} from '@umb-network/toolbox';
 import {PayableOverrides} from '@ethersproject/contracts';
 
+import chainV470abi from './ChainV4.7.0.abi.json';
```

Comment:
Do we need to store various versions of the abi in the codebase? Why not just `Chain.abi.json` ?

Reply:
done:
`import chainAbi from './Chain.abi.json';`